firestore.rules

```diff
@@ -0,0 +1,29 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    
+    function isAuthenticated() {
+    	return request.auth != null && request.auth.token.email_verified == true;
+    }
+
+    function isAdmin() {
+      return request.auth.token.isAdmin == true;
+    }
+    
+
+    match /users/{userId} {
+      
+    function isUser() {
```

Comment:
can we indent this function properly, use ES6 syntax and name it `isIdentified` ?